ProjFS.Mac/PrjFSKext/KauthHandlerTestable.hpp

```diff
@@ -12,5 +13,20 @@
 
 KEXT_STATIC_INLINE bool FileFlagsBitIsSet(uint32_t fileFlags, uint32_t bit);
 KEXT_STATIC_INLINE bool ActionBitIsSet(kauth_action_t action, kauth_action_t mask);
+KEXT_STATIC_INLINE bool TryGetFileIsFlaggedAsInRoot(vnode_t vnode, vfs_context_t context, bool* flaggedInRoot);
 KEXT_STATIC bool IsFileSystemCrawler(const char* procname);
 KEXT_STATIC bool ShouldIgnoreVnodeType(vtype vnodeType, vnode_t vnode);
+KEXT_STATIC bool ShouldHandleVnodeOpEvent(
+                                          // In params:
```

Comment:
Thanks for bringing this up.

When I autoformat in XCode (Cntrl-I), it does this level of indenting.
I really don't like fighting autoformatting of editors in new code.

Am I correct that this is the autoformat of XCode?  If not, please advise me on how to fix.  If so, can we consider moving to autoformat standards for new code?  @wilbaker

Reply:
Xcode's default indentation rules are unfortunately optimised for Objective-C's named function parameters and tend to cause excessive indentation for C and C++ style function calls. I personally have "Automatically indent based on syntax" disabled (Preferences -> Text Editing -> Indentation); as far as I'm aware, this is the only way to turn off the crazy levels of function argument indentation. Unfortunately, unlike indentation level rules (spaces/tabs, number of spaces) this isn't something you can set on a per-file or per-project basis.

I don't think that level of indentation is in any way helpful, so I'm not keen on changing our style guidelines simply because Xcode isn't designed particularly well for C or C++ editing.